roles/idrac_user/tasks/get_user.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Fetch user accounts
+  ansible.builtin.uri:
+    url: "https://{{ hostname }}:{{ https_port }}/redfish/\
```

Comment:
pass ipwrap filter for hostname, otherwise will fail.

Reply:
I had went through the module, it was no where mentioned that the module supports IPv6. So didn't use.